src/patternfly/components/Hint/hint.scss

```diff
@@ -0,0 +1,114 @@
+.pf-c-hint {
+  --pf-c-hint__header--GridColumnWidth: auto;
+  --pf-c-hint__actions--GridColumnWidth: calc(var(--pf-global--spacer--2xl) + #{pf-size-prem(44px)});
+  --pf-c-hint--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-hint--BackgroundColor: var(--pf-global--palette--blue-50);
+  --pf-c-hint--BorderColor: var(--pf-global--palette--blue-100);
+  --pf-c-hint--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-hint--BoxShadow: var(--pf-global--BoxShadow--sm);
+  --pf-c-hint--Color: var(--pf-global--Color--100);
+
+  // Hint Header
+  --pf-c-hint__header--FontSize: var(--pf-global--FontSize--lg);
```

Comment:
In keeping with the other components, `__header` designates a region (ie, `<header>`), and `__title` is the title (ie, `<h1>`). If we're setting `font-size`, I would expect this to be a title element, like the alert (which this layout seems to be modeled after).

Reply:
great point, updated header to be title